one_sequence[] robot_sequences
